scripts/validate-version-bump.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -eux
```

Comment:
Why remove this? If you want to remove the logging, that's just the x. Also, if you're removing any part of this, please update the `trap` accordingly

Reply:
I added back eu and updated the trap.  Removing the x was to quiet the logs a bit; I had assumed it was added there to sanity check that the script was working as expected.  I'm not a bash expert, though...

Reply:
I like -x because it creates a useful debugging error trail when things go wrong unexpectedly, but it does seem to be off by default at Citrine.